deploy/docker/Dockerfile

```diff
@@ -43,7 +43,9 @@ RUN gem sources --clear-all \
 # Default settings for log collection
 ENV LOG_FORMAT "fields"
 ENV FLUSH_INTERVAL "5s"
-ENV NUM_THREADS "1"
+ENV NUM_THREADS "4"
```

Comment:
Does it mean every plugin will have 4 threads? Does it harm if we set it to an even higher value?

Reply:
Yes it does.  I picked 4 because that's what the collector uses.  I recommend we try it for a while, work on instrumenting some metrics so we can see queue size, and only increase it once we can observe the effects.